application/src/main/resources/org/opentripplanner/apis/gtfs/schema.graphqls

```diff
@@ -1801,6 +1801,10 @@ type RealTimeEstimate {
 type RentalVehicle implements Node & PlaceInterface {
   "If true, vehicle is currently available for renting."
   allowPickupNow: Boolean
+  "Fuel or battery power remaining in the vehicle. Expressed from 0 to 1."
+  currentFuelPercent: Float
+  "Range in meters that the vehicle can travel with the vehicle's current charge or fuel."
+  currentRangeMeters: Float
```

Comment:
I want to explore if there is a way to express a general distance as a string, just like a duration has the `PT30M` format. I'm not sure if it's worth it and if the others agree so lets talk about it in the dev meeting.

Internally, we could also use the `Distance` class which is currently in a sandbox.

Reply:
Yeah we can talk about it, but I can't participate in this weeks meetings. I would appreciate it if you wait for me in the next week meeting.
In the meantime I'll take a look at the Distance class and the format of duration as a reference

Reply:
Is `utils/distance/Distance.java` a good place for Distance class? 

I implemented the Distance class for the range, and thought about the format you mentioned and in my opinion we should stick with a numeric value:
* [This post](https://stackoverflow.com/questions/27831690/java-duration-like-class-for-length), and the link attached to it, explains why there is a Duration class with a more complex representation  and I don't think that can be applied to distance
* I think we should stick to meters because of the GBFS standard uses that and it is more simple, a more complex representation with maybe more units would create confusion
* The conversion from other units (km, miles...) to meters is simple so even in a query it would be simple for a frontend app to make it, on the other hand converting a string with multiple units to meters it would be more cumbersome

Reply:
I think there is no good standard for this format so I hesitantly agree with you there. 

The Distance class should be moved to `org.opentripplanner.transit.model.basic` and could you please add a few test cases.